scapy/layers/vrrp.py

```diff
@@ -62,9 +62,16 @@ class VRRPv3(Packet):
         BitField("res", 0, 4),
         BitField("adv", 100, 12),
         XShortField("chksum", None),
-        # FIXME: addrlist should also allow IPv6 addresses :/
-        FieldListField("addrlist", [], IPField("", "0.0.0.0"),
-                       count_from=lambda pkt: pkt.ipcount)]
+        # use addrlist for IPv4, addrlist6 for IPv6
```

Comment:
Can you use `addrlist` for both IPv4 and IPv6?

Reply:
The PR is updated. IPv4 and IPv6 addresses both use the addrlist field now.